modules/billing/main.tf

```diff
@@ -0,0 +1,95 @@
+resource "google_billing_budget" "budget" {
```

Comment:
Can the calling of this module be simplified? There are references to variables that are not of interest to us currently. See [YAGNI](https://github.com/webpro/programming-principles#yagni) and [Do The Simplest Thing That Could Possibly Work](https://github.com/webpro/programming-principles#do-the-simplest-thing-that-could-possibly-work).